validator/validator-main.protoascii

```diff
@@ -26,7 +26,7 @@ min_validator_revision_required: 328
 # newer versions of the spec file. This is currently a Google internal
 # mechanism, validator.js does not use this facility. However, any
 # change to this file (validator-main.js) requires updating this revision id.
-spec_file_revision: 662
+spec_file_revision: 658
```

Comment:
spec file revision shouldn't change

Reply:
i didn't change intentionally. Not sure how this happened. reverted.